src/app/services/services.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
```

Comment:
I don't think we should be grouping all services in this way - angular 2+ makes it easy to group stuff by domain, so we should include the services wherever they are most relevant, not in a giant common directory.  The main raster foundry repo is a good example of how we don't want our services to end up.